async-opcua-types/src/byte_string.rs

```diff
@@ -250,6 +250,16 @@ impl ByteString {
         }
     }
 
+    /// Creates a byte string from a Base64 encoded string, ignoring whitespace.
```

Comment:
Nitpicking (noticed the same capitalized version a few lines down, but I don't think this should be capitalized):
```suggestion
    /// Creates a byte string from a base64 encoded string, ignoring whitespace.
```